diff -up gpgme-1.17.0/configure.ac.omv~ gpgme-1.17.0/configure.ac
--- gpgme-1.17.0/configure.ac.omv~	2022-02-23 22:27:57.940694261 +0100
+++ gpgme-1.17.0/configure.ac	2022-02-23 22:28:12.250743898 +0100
@@ -425,7 +425,7 @@ if test "$found_py" = "1"; then
 	if test "$found_py" = "1" -o "$found_py3" = "1"; then
 	  # Reset everything, so that we can look for another Python.
           m4_foreach([mym4pythonver],
-                     [[2.7],[3.4],[3.5],[3.6],[3.7],[3.8],[3.9],[all]],
+                     [[2.7],[3.4],[3.5],[3.6],[3.7],[3.8],[3.9],[3.10],[3.11],[all]],
            [unset PYTHON
 	    unset PYTHON_VERSION
 	    unset PYTHON_CPPFLAGS
diff -up gpgme-1.17.0/m4/python.m4.omv~ gpgme-1.17.0/m4/python.m4
--- gpgme-1.17.0/m4/python.m4.omv~	2022-02-24 00:25:24.808792990 +0100
+++ gpgme-1.17.0/m4/python.m4	2022-02-24 00:27:55.854400618 +0100
@@ -42,7 +42,7 @@ AC_DEFUN([AM_PATH_PYTHON],
   m4_define_default([_AM_PYTHON_INTERPRETER_LIST],
 [python2 python2.7 dnl
  python dnl
- python3 python3.9 python3.8 python3.7 python3.6 python3.5 python3.4
+ python3 python3.11 python3.10 python3.9 python3.8 python3.7 python3.6 python3.5 python3.4
  ])
 
   AC_ARG_VAR([PYTHON], [the Python interpreter])
@@ -94,7 +94,7 @@ AC_DEFUN([AM_PATH_PYTHON],
   dnl library.
 
   AC_CACHE_CHECK([for $am_display_PYTHON version], [am_cv_python_version],
-    [am_cv_python_version=`$PYTHON -c "import sys; sys.stdout.write(sys.version[[:3]])"`])
+    [am_cv_python_version=`$PYTHON -c "import sys; print(str(sys.version_info.major) + '.' + str(sys.version_info.minor))"`])
   AC_SUBST([PYTHON_VERSION], [$am_cv_python_version])
 
   dnl Use the values of $prefix and $exec_prefix for the corresponding
