--- gpgme-1.8.0/lang/cpp/src/GpgmeppConfig.cmake.in.in.omv~	2016-11-20 23:07:44.903144901 +0100
+++ gpgme-1.8.0/lang/cpp/src/GpgmeppConfig.cmake.in.in	2016-11-20 23:08:24.566215338 +0100
@@ -63,7 +63,7 @@ add_library(Gpgmepp SHARED IMPORTED)
 
 set_target_properties(Gpgmepp PROPERTIES
   INTERFACE_INCLUDE_DIRECTORIES "@resolved_includedir@/gpgme++;@resolved_includedir@"
-  INTERFACE_LINK_LIBRARIES "pthread;@resolved_libdir@/libgpgme@libsuffix@;@LIBASSUAN_LIBS@"
+  INTERFACE_LINK_LIBRARIES "pthread;-lgpgme;@LIBASSUAN_LIBS@"
   IMPORTED_LOCATION "@resolved_libdir@/libgpgmepp.so"
 )
 
